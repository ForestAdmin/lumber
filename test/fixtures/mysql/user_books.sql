CREATE TABLE user_books (
	user_id int NOT NULL,
	book_id int NOT NULL,
	checkout_date date,
	return_date date,
	CONSTRAINT user_books_pkey PRIMARY KEY (user_id, book_id),
	CONSTRAINT user_books_book_id_fkey FOREIGN KEY (book_id) REFERENCES books(id) ON DELETE NO ACTION ON UPDATE CASCADE,
	CONSTRAINT user_books_user_id_fkey FOREIGN KEY (user_id) REFERENCES users(id) ON DELETE NO ACTION ON UPDATE CASCADE
);