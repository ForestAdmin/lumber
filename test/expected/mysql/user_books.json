{
  "fields":[
    {
      "name":"checkoutDate",
      "nameColumn":"checkout_date",
      "type":"DATE",
      "primaryKey":false,
      "defaultValue":null
    },
    {
      "name":"returnDate",
      "nameColumn":"return_date",
      "type":"DATE",
      "primaryKey":false,
      "defaultValue":null
    }
  ],
  "references":[],
  "primaryKeys":["user_id","book_id"],
  "options":{
    "underscored":true,
    "timestamps":false,
    "hasIdColumn":false,
    "hasPrimaryKeys":true
  }
}